.php_cs

```diff
@@ -1,13 +1,18 @@
 <?php
 
-/*
- * In order to make it work, fabpot/php-cs-fixer and sllh/php-cs-fixer-styleci-bridge must be installed globally
- * with composer.
- *
- * @link https://github.com/Soullivaneuh/php-cs-fixer-styleci-bridge
- * @link https://github.com/FriendsOfPHP/PHP-CS-Fixer
- */
+$config = PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
+    ->setRules([
+        '@Symfony' => true,
+        '@Symfony:risky' => true,
+        'array_syntax' => ['syntax' => 'short'],
```

Comment:
wasn't this one part of the `@Symfony` ruleset already?

![image](https://user-images.githubusercontent.com/2707563/70327876-1226f680-1838-11ea-82d6-54e52d53ace2.png)

see https://github.com/FriendsOfPHP/PHP-CS-Fixer

Reply:
It looks like the documentation is not up to date: *defaults to 'long'*
But you're right, this ruleset is part of `@Symfony` with `syntax: short`.
I remove this line, thanks.